examples/tutorial/tutorial_1.py

```diff
@@ -30,6 +30,7 @@
 
 with CorpusContext(corpus_name) as c:
     q = c.query_graph(c.phone).group_by(c.phone.label.column_name('phone'))
+    q = q.order_by(c.lexicon_phone.label)
```

Comment:
add a comment that `order_by` isn't strictly necessary, but will make the outputted CSV cleaner.